packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposal/acceptProposal.ts

```diff
@@ -16,10 +22,61 @@ type ContractInput = {
 
 const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInput> => {
   if (flags.input) return flags.input as CommandInput
+  const { rdd: rddPath, secret } = flags
+
+  if (!rddPath) throw new Error('RDD flag is required. Provide it with --rdd flag')
+
+  const rdd = RDD.getRDD(rddPath)
+  const contract = args[0]
+
   return {
     proposalId: flags.proposalId,
     digest: flags.digest,
+    offchainConfig: getOffchainConfigInput(rdd, contract),
+    randomSecret: secret,
+  }
+}
+
+const beforeExecute: BeforeExecute<CommandInput, ContractInput> = (context) => async () => {
+  const { proposalId, randomSecret, offchainConfig: offchainLocalConfig } = context.input
+
+  const { offchainConfig } = await serializeOffchainConfig(offchainLocalConfig, process.env.SECRET!, randomSecret)
+  const localConfig = offchainConfig.toString('base64')
+
+  // Config in Proposal
+  const proposal: any = await context.provider.wasm.contractQuery(context.contract, {
+    proposal: {
+      id: proposalId,
+    },
+  })
+  const offchainConfigInProposal = await deserializeConfig(Buffer.from(proposal.offchain_config, 'base64'))
+  const configInProposal = longsInObjToNumbers({
+    ...offchainConfigInProposal,
+    offchainPublicKeys: offchainConfigInProposal.offchainPublicKeys?.map((key) => Buffer.from(key).toString('hex')),
+    f: proposal.f,
+  })
+
+  try {
+    assert.equal(localConfig, proposal.offchain_config)
+  } catch (err) {
+    throw new Error('RDD configuration does not correspond the proposal configuration')
   }
+
+  logger.success('RDD Generated configuration matches with onchain proposal configuration')
```

Comment:
This validation should be moved at LOC:52 before it's actually used.

Also, consider adding `err.message` from the source error to the new wrapped error thrown, so we perhaps get some more info on the output about the differences. I know it's `base64` encoded, but may be useful to debug sometimes.